schemas/client.yml

```diff
@@ -59,19 +59,35 @@
       description: >
         Client domain.
 
+    - name: name
```

Comment:
What is this field's relationship to `client.domain`?

Reply:
domain is listed under host as being e.g. the windows domain (e.g. netbios name - "CONTOSO");  name is used elsewhere (dns.question.name) as being specific to the domain name...

Up until digging thru this I had expected to lobby for fqdn, but based on the fact that its next to impossible to determine from information gathered about a dns request if a name is partially or fully qualified, I went with the DNS spec as the baseline for consistency

Reply:
@webmat - did I answer the question?  I'm not 100% sure I understood your original question (as I looked at the other references)